Jenkinsfile

```diff
@@ -68,80 +76,59 @@ spec:
                 }
             }
         }
-        stage('Publish Packages to Registry') {
+        stage('Publish') {
             when {
                 branch 'dev'
             }
             steps {
                 container('node14') {
                     script {
-                        // echo 'Publishing packages to private NPM registry...'
-                        // sh 'echo "registry=https://registry.web.asu.edu/" > ~/.npmrc'
-                        // sh 'echo "always-auth=true" >> ~/.npmrc'
-                        // sh 'echo "//registry.web.asu.edu/:_authToken=$NPM_TOKEN" >> ~/.npmrc'
-                        echo '# Publishing packages to GitHub Packages...'
+                        // TODO Remove after transition as it will be set in environment block:
+                        // Use Github token as NPM token with GH Packages
+                        NPM_TOKEN = GH_TOKEN
 
-                        echo '## Configuring .npmrc file...'
+                        echo '# Publishing packages to GitHub Packages registry...'
+
+                        echo '## Re-set .npmrc file for Github Package registry...'
+                        // Note: In the first command, the single > redirect (re)creates file
                         sh 'echo "@asu:registry=https://npm.pkg.github.com" > ~/.npmrc'
                         sh 'echo "always-auth=true" >> ~/.npmrc'
                         sh 'echo "//npm.pkg.github.com/:_authToken=$GH_TOKEN" >> ~/.npmrc'
-                      
-                        sh 'echo $GH_TOKEN'
-                        sh 'echo ${GH_TOKEN}'
-                        sh 'cat ~/.npmrc'
 
                         echo '## Publishing packages...'
-                        try {
-                          sh 'yarn publish-packages'
-                        } catch (e) {
-                          echo '### Publishing packages failed...'
-                          echo "Error: ${e}"
-                        }
+                        sh 'yarn publish-packages'
                     }
                 }
             }
         }
-        stage('Deploy Storybook to GitHub Pages') {
+        stage('Deploy') {
             when {
-                branch 'dev'
+              branch 'dev'
             }
             steps {
                 container('node14') {
                     script {
-                        sh 'yarn add @storybook/storybook-deployer --ignore-workspace-root-check --registry https://registry.npmjs.org'
+                        // TODO Remove after transition as it will be set in environment block:
+                        // Use Github token as NPM token with GH Packages
+                        NPM_TOKEN = GH_TOKEN
+
+                        // echo "Debug persisted .npmrc..."
+                        // sh 'cat ~/.npmrc'
+                        // sh 'yarn config list'
+
+                        echo '# Final, post-publish install and build...'
                         sh 'yarn install'
```

Comment:
I understood that we don't need to run again the install and build commands. Is this ok?